OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -67,21 +70,39 @@ RUN yum install -y \
         gcc gcc-c++ glibc glibc-devel libaio libaio-devel libgcc \
 	libstdc++ libstdc++-devel ksh make ocfs2-tools sysstat && \
     yum clean all && \
-    rm -rf /var/cache/yum/* && \
     chmod a+xr /u01 && \
     useradd -b /u01 -m -s /bin/bash oracle && \
     echo oracle:oracle | chpasswd && \
```

Comment:
You are hardcoding a password for the `oracle` user which is against security policy. I don't see any reason why you're setting a password anyway. There is no way to remotely login as the `oracle` user.